docs/examples/managed/ColBERT_demo.ipynb

```diff
@@ -0,0 +1,689 @@
+{
```

Comment:
<p>looks there is a special token in the end</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/10477/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
let me remove it.